crates/control/src/behavior/defend.rs

```diff
@@ -158,6 +283,11 @@ fn defend_goal_pose(
                     kicking_team: Team::Opponent,
                 },
             ..
+        })
+        | Some(GameControllerState {
```

Comment:
How can I do this since one uses game_phase and the other uses sub_state? Does this work with some kind of bitwise or | operation?

Reply:
Ive merged these so that only one Some() statement exists, but don't know how to merge it further since one case uses sub_state and the other uses game_phase